createdisk-library.sh

```diff
@@ -177,6 +177,7 @@ function generate_hyperkit_directory {
     local kernel_release=$4
     local kernel_cmd_line=$5
 
+    mkdir "$destDir"
```

Comment:
Shouldn't the function also renamed to `generate_hyperkit_bundle` instead `generate_hyperkit_directory` here?

Reply:
ah thanks, forgot to change it :)